client/pages/signup.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+
+export default class SignUp extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: null,
+      passError: null
+    };
+
+    this.submit = this.submit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  submit(e) {
+    e.preventDefault();
+    const form = document.forms[0];
+    const data = {
+      firstName: form.firstName.value,
+      lastName: form.lastName.value,
+      email: form.email.value,
+      password: form.password.value
+    };
```

Comment:
In React we want to avoid as much as possible directly interacting with the DOM through the `document` object. You should be getting your form values from `this.state`